bin/tictactoe

```diff
@@ -1,3 +1,26 @@
 #!/usr/bin/env ruby
-
 require_relative '../config/environment'
+
+  puts "welcome to tic tac toe!"
+  puts "what kind of game would you like to play, 0,1, or player."
+num_players = gets.strip
+if num_players.to_i == 1
+puts "only 1"
+puts "Who wants to go first? 1 for X or 2 for O."
+first = gets.strip
+if first.to_i == 1
+game = Game.new(Players::Human.new("X"),Players::Computer.new("0"))
+game.play
+elsif first.to_i == 2
+game = Game.new(Players::Computer.new("X"),Players::Human.new("O"))
+game.play
+else
+puts "invalid entry - Please start again."
+end
+elsif num_players.to_i == 2
+puts "only 2"
+game = Game.new(Players::Computer.new("X"),Players::Human.new("O"))
+game.play
+elsif num_player.to_i == 0
+puts "0 players"
+end
```

Comment:
Prefer to encapsulate CLI logic into a class. For example, the only code in your bin executable should be:

```ruby
#!/usr/bin/env ruby
       
require_relative '../config/environment'

TicTacToeCLI.new.call
```

And then you'd have a file: `lib/tic_tac_toe_cli.rb` with:

```ruby
class TicTacToeCLI
  def initialize
  end

  def call
   # Your game logic
  end
end
```

The code in your bin would generally just go in `call`.

Also, pay attention to your indenting.  Ruby convention is a two-space indent after any class declaration, method, or block.  You can check out this great [Ruby style guide](https://github.com/bbatsov/ruby-style-guide)!